omod/src/main/java/org/openmrs/module/owa/utils/OwaFileNameUtil.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0 + Health disclaimer. If a copy of the MPL was not distributed with
+ * this file, You can obtain one at http://license.openmrs.org
+ */
+package org.openmrs.module.owa.utils;
+
+import org.apache.commons.io.FilenameUtils;
+import org.apache.commons.lang.StringUtils;
+
+/**
+ * This class contains methods that remove the version number from the file name when installing an
+ * OWA
+ */
+public class OwaFileNameUtil {
+
+    /**
+     * This method gets the file name from the OWA installation url that is passed to it
```

Comment:
Proper javadoc also documents the parameters and return value

Reply:
I have added the parameters and return value to the doc string